utils/githooks/prepare-commit-msg.d/10-submodule-update-check

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -eu
+
+python utils/githooks/submodule_update_check.py "$1"
```

Comment:
Could we use python3, or is that similar to the other reason why we still have some python2 code?